### ISSUE TITLE
Inconsistencies in the metadata file

### ISSUE THREAD

**Issue by fplazaonate** [yue-clare-lou/Well2WellContaminationPublication#1]
Hi,

I have noticed some inconsistencies in the metadata files:
1. sample_name "72D17" is duplicated in both NCBI_code_name2sample_name.csv and P3_plate_map.csv
2. NCBI_sample_code is missing for mothers
3. metadata for samples L1_XXXX available in the NCBI SRA are missing

Could you fix this?

Thanks in advance,
Florian

**Comment by yue-clare-lou** [comment]
Hi Florian,

Thanks for the message.

- Regarding your points 1 & 2, I just updated NCBI_code_name2sample_name.csv and P3_plate_map.csv to address your issues. I apologize for this.
- For your point 3, as indicated in our manuscript, 14 samples (NCBI sample names L1_XXX_XXX) were extracted via single tubes. I just added an "additional_notes" file to the case_study_I folder explaining this.

Let me know if you have any more questions!

Best,
Clare
